rest/taskrouter/workflows/list/post/example-1/example-1.5.x.rb

```diff
@@ -0,0 +1,44 @@
+# Get twilio-ruby from twilio.com/docs/ruby/install
+require 'rubygems'          # This line not needed for ruby > 1.8
+require 'twilio-ruby'
+
+# Get your Account Sid and Auth Token from twilio.com/user/account
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+workspace_sid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+
+client = Twilio::REST::Client.new account_sid, auth_token
+
+sales_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+marketing_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+support_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+everyone_queue_sid = "WQXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+
+
+config = {
+    'task_routing': {
+        'filters': [
+            {
+                'expression': "type=='sales'",
+                'targets': [{'queue': sales_queue_sid}]
+            },
+            {
+                'expression': "type=='marketing'",
+                'targets': [{'queue': marketing_queue_sid}]
+            },
+            {
+                'expression': "type=='support'",
+                'targets': [{'queue': support_queue_sid}]
+            }
+        ],
+        'default_filter': {
+            'queue': everyone_queue_sid
+        }
+    }
+}
+
+
+json = config.to_json()
+
+workflow = client.taskrouter.v1.workspaces(workspace_sid).workflows.create(friendly_name: 'Sales, Marketing, Support Workflow', assignment_callback_url: 'http://example.com', fallback_assignment_callback_url: 'http://example.2com', task_reservation_timeout: '30', configuration: json)
```

Comment:
I would clean up some of the really long lines to keep them under 100 chars in length. What do you think @juancarlospaco ?